muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml

```diff
@@ -12770,7 +12771,7 @@ components:
 
     RecordWorkspaceActivityInfo:
       type: object
-      description: RecordsWorkspaceActivityInfo object
+      description: Record workspace activity info with line and category, default line and activities
       required:
         - lineName
         - lineCategory
```

Comment:
These are not "lines", these are studyPrograms

Reply:
Rest speaks about lines.

Reply:
I'll make an issue about this. Carry on.